### PR TITLE
chore: format readme and add note about 4.0 beta

### PR DESCRIPTION
**Pull request by nbbeeken**
Added a small note at the top of the readme about 4.0 beta, let me know if we want to make it more noticable somehow

(Also ran prettier on the readme which makes the code examples match our code style)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
-[![npm](https://nodei.co/npm/mongodb.png?downloads=true&downloadRank=true)](https://nodei.co/npm/mongodb/) [![npm](https://nodei.co/npm-dl/mongodb.png?months=6&height=3)](https://nodei.co/npm/mongodb/)
+# MongoDB NodeJS Driver
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mongodb/node-mongodb-native?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-
-# Description
+[![npm](https://nodei.co/npm/mongodb.png?downloads=true&downloadRank=true)](https://nodei.co/npm/mongodb/)
 
 The official [MongoDB](https://www.mongodb.com/) driver for Node.js. Provides a high-level API on top of [mongodb-core](https://www.npmjs.com/package/mongodb-core) that is meant for end users.
```
Comment (kmahar) [inline README.md:5]: mention of core seems irrelevant now that we merged the repos, want to take that out?

File: README.md [hunk 2]
```diff
@@ -64,10 +65,10 @@ You can also use the [Yarn](https://yarnpkg.com/en) package manager.
 
 The MongoDB driver depends on several other packages. These are:
 
-* [mongodb-core](https://github.com/mongodb-js/mongodb-core)
-* [bson](https://github.com/mongodb/js-bson)
-* [kerberos](https://github.com/mongodb-js/kerberos)
-* [node-gyp](https://github.com/nodejs/node-gyp)
+- [mongodb-core](https://github.com/mongodb-js/mongodb-core)
```
Comment (kmahar) [inline README.md:68]: also outdated I think? this seems maybe a bit oudated in general?

**Review by kmahar — COMMENTED** [review]
your changes look pretty good but I'm looking through the whole README and a lot of it seems very old and probably outdated? doesn't have to be done right now but I think before 4.0 GA we should try to tidy this up a lot

**Review by kmahar — APPROVED** [review]
lgtm, could you please open a ticket about a more thorough editing before 4.0 GA?

File: README.md [hunk 3]
```diff
@@ -196,7 +197,7 @@ const dbName = 'myproject';
 // Use connect method to connect to the server
 MongoClient.connect(url, function(err, client) {
```
Comment (durran) [inline README.md:198]: Would we want to change the example to use the `MongoClient` constructor instead? (Just to get people in the habit of doing that since if they want to listen to events from the very beginning, like SDAM events, they will need to register the handlers before connecting)

**Review by durran — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (nbbeeken) [reply README.md:198]: SGTM

**Review by durran — APPROVED** [review]
LGTM. Thanks!

File: README.md [hunk 4]
```diff
@@ -41,10 +42,10 @@ Change history can be found in [`HISTORY.md`](HISTORY.md).
...
 
- * [MongoDB](https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#reference-compatibility-mongodb-node)
- * [NodeJS](https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#reference-compatibility-language-node)
+- [MongoDB](https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#reference-compatibility-mongodb-node)
+- [NodeJS](https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#reference-compatibility-language-node)
 
-# Installation
+## Installation
 
 The recommended way to get started using the Node.js 3.0 driver is by using the `npm` (Node Package Manager) to install the dependency in your project.
```
Comment (emadum) [inline README.md:50]:
```suggestion
The recommended way to get started using the Node.js driver is by using `npm` (Node Package Manager) to install the dependency in your project.
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mongodb/node-mongodb-native?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![npm](https://nodei.co/npm/mongodb.png?downloads=true&downloadRank=true)](https://nodei.co/npm/mongodb/)
 
-# Description
+The official [MongoDB](https://www.mongodb.com/) driver for Node.js.
 
-The official [MongoDB](https://www.mongodb.com/) driver for Node.js. Provides a high-level API on top of [mongodb-core](https://www.npmjs.com/package/mongodb-core) that is meant for end users.
+**Looking for the latest?** We're working on the next major version of the driver now in beta.
+Check out our [beta version 4.0 here](https://github.com/mongodb/node-mongodb-native/tree/4.0).
```
Comment (emadum) [inline README.md:8]:
```suggestion
### Version 4.0

**Looking for the latest?** We're working on the next major version of the driver, now in beta.
Check out our [beta version 4.0 here](https://github.com/mongodb/node-mongodb-native/tree/4.0), which includes a full migration of the driver to TypeScript.
```

I think adding a heading and mentioned TypeScript would help draw attention to the beta.

**Review by emadum — COMMENTED** [review]
LGTM! Just had a few suggestions.

**Review by emadum — APPROVED** [review]
LGTM 👍
